test/fail_compilation/badimport.d

```diff
@@ -0,0 +1,4 @@
+// COMPILED_IMPORTS: imports/nomodname.d
+// REQUIRED_ARGS: -Ifail_compilation
+// PERMUTE_ARGS:
+import imports.nomodname;
```

Comment:
no newline ad EOF

Reply:
Is that a bad thing?

Reply:
i think it's a convention (lots of editor show an annoying marker when we forget one